src/replacements/gcn-circular/test.md

```diff
@@ -13,3 +13,91 @@ GCN Circ. 789 and GCN Circs. 3 and 4
+
+This one has parenthesis, a capital c, a period, and a hashtag (GCN Circ. #123),
+
+This one has a capital c and no other punctuation GCN Circ 123.
+
+This one has parenthesis, a capital c and no other punctuation (GCN Circ 123)
+
+This one has a full word circular, a capital c, and no other punctuation. GCN Circular 123
+
+This has a capital C and a hashtag GCN Circular #123
+
+This one has parenthesis, a capital c, a full word circular and no hashtag (GCN Circular 123).
+
+This one (GCN Circular #123) has parenthesis, a capital c and a hashtag.
+
+This one GCN Circs 123, 456 has two circulars, no parenthesis (but heres a set just because), and no hashtag.
+
+This one has two circs, no parenthesis, GCN Circs. 123, 456 is comma separated, c is capital and word is abbreviated.
+
+This one has the same as above but without periods GCN Circs #123, #456 and with hashtags. #bonustag
+
+This one has parenthesis, no period, hashtags, and a comma separateding it (GCN Circs #123, #456).
+
+This one has parenthesis, comma, no period, a capital c, and no # (GCN Circs 123, 456).
+
+This GCN Circ 123, 456 has a capital c, abbreviated word, no period, no hashtag, and no parenthesis.
+
+This one has a capital c, a period, comma separated, GCN Circ. 123, 456 and no 42 (0) parenthesis.
+
+This one has capital c, hashtags, and abbreviated word GCN Circ #123, #456. It also has no parenthesis.
+
+This one has parenthesis (GCN Circ 123, 456) and in those (bonus parenthesis test) no hashtag, a comma, and no period.
+
+This is lower case, abbreviated, no hashtags, and comma separated GCN circs 123, 456.
+
+This one has lower case c, a period, no parenthesis, a comma, and no # GCN circs. 123, 456.
+
+This one has a lower case c and hashtags comma separated GCN circs #123, #456. more, commas, for, testing, 123, 456
+
+This one has parenthesis, a lower case c, no hashtag, and a comma (GCN circs 123, 456).
+
+This just has the word circs. 42 circulars. a circ 23 feet across. #0000.
+
+This one has a lower case c, no hashtags, no parenthesis, and an and GCN circs 123 and 456.
+
+This one has no parenthesis, a lowercase c, a period, no hashtags, and an and GCN circs. 123 and 456.
+
+This onw has a lowercase circs, a hashtag, an and, and no parenthesis GCN circs #123 and #456!
+
+This one has parenthesis (GCN circs 123 and 456) a lowercase circs, no period, no hashtags, and an and.
+
+This one has no punctuation and just a space separating it. GCN circs 123 456 and the circs is lower case.
+
+GCN circs. 123 456 has a lowercase circs with a period, no hashtags, no parenthesis, and only a space between ids.
+
+This one is similar but with hashtags GCN circs #123 #456
+
+And (GCN circs 123 456) has parenthesis, no hashtags, no period, and a space between two ids.
+
+This one is all lowercase gcn circ 123, 456
+
+This one has some spacing issues and a hashtag GCN #123
+
+This one just has spacing issues gcn 123.
```

Comment:
It might be more legible if you actually omit the description text for each line, as these are all minor variations of the same tests:

```suggestion
All of these minor variations with parentheses, hash signs, and periods are also supported:
GCN 123
GCN #123
(GCN 123)
(GCN #123)
GCN circ. 123
GCN circ. #123
(GCN circ. #123)
GCN circ 123
(GCN circ 123)
GCN Circ. 123
GCN Circ. #123
(GCN Circ. #123)
GCN Circ 123
(GCN Circ 123)
GCN Circular 123
GCN Circular #123
(GCN Circular 123)
(GCN Circular #123)
GCN Circs 123, 456
GCN Circs. 123, 456
GCN Circs #123, #456
(GCN Circs #123, #456)
(GCN Circs 123, 456)
GCN Circ 123, 456
GCN Circ. 123, 456
GCN Circ #123, #456
(GCN Circ 123, 456)
GCN circs 123, 456
GCN circs. 123, 456
GCN circs #123, #456
(GCN circs 123, 456)
GCN circs 123 and 456
GCN circs. 123 and 456
GCN circs #123 and #456
(GCN circs 123 and 456)
GCN circs 123 456
GCN circs. 123 456
GCN circs #123 #456
(GCN circs 123 456)
GCN          #123
gcn         123
```